games-strategy/dunelegacy/dunelegacy-0.96.4_22.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="A Dune II game engine"
+DESCRIPTION="Dune Legacy is an effort by a handful of developers to revitalize \
+the first-ever real-time strategy game. \
+It tries to be as similar as possible to the original gameplay but to integrate \
+user interface features most modern realtime-strategy games have like selecting \
+multiple units"
+HOMEPAGE="https://dunelegacy.sourceforge.net/website/"
+COPYRIGHT="2016-2022 Dune Legacy"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/code-snapshots/git/d/du/dunelegacy/code.git/dunelegacy-code-6ea9ac96854daa8c75ba429e78dc6716b147e106.zip"
```

Comment:
You now have 2 different recipes, do they need to co-exist?
Also, I think this is a snapshot/commit checkout? In that case `srcGitRev` is preferable.

Reply:
Not really, the new version seems better in every regard ngl

Reply:
Actually, I'm encountering an issue while downloading this version, the download link has to be generated first so it doesn't actually work in the recipe :(

Reply:
Fixed